UPGRADE-2.0.md

```diff
@@ -286,6 +291,39 @@
   level of the event. This has been changed so that only the `ErrorException` or its
   derivated classes are considered for this behavior.
 
+- The method `Client::createProcessors` has been removed as there is no need to create
+  instances of the processors from outside the `Client` class.
+
+- The method `Client::setProcessors` has been removed. You should use `Client::addProcessor`
+  and `Client::removeProcessor` instead to manage the processors that will be executed.
+
+  Before:
+
+  ```php
+  $processor1 = new Processor();
```

Comment:
You should make this example more clear: you forgot to mention (and add to this example) that `Processor` was an abstract class that was to be extended, that you replaced with the `ProcessorInterface`.

Reply:
`Processor` here was meant to be any processor, not the specific abstract class. Maybe I could use existing processor names or make it more clear using something like `FooProcessor`?